README.md

```diff
@@ -1,2 +1,110 @@
+ - [Prerequisites](#prerequisites)
+ - [Install](#install)
+ - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+- 
+<!-- PROJECT DESCRIPTION -->
+# 📖 [Hello Microverse] <a name="about-project"></a>
+> This is my first microverse project to show how I use tools.
+**Hello Microverse** is a great project to learn about Linters.
+This project is compiled using HTML, CSS and JS.
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+- **It has MIT License**
+- **It has HTML files**
+- **It has CSS files**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A code editor of your choice (like VS code or atom or any other)
+- Version Control System (Git is preferred)
+
+```sh
+  npm install
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/Nomarg/Hello-Microverse.git
+  cd Hello-Microverse
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Hello-Microverse
+  gem install
+```
+### Usage
+To run the project, execute the following command:
+```sh
+ rails server
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+👤 **Author1**
+- GitHub: [@AdrianScharff](https://github.com/AdrianScharff)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/adrián-juárez-17a51b263)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **Javascript files**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## ⭐ Show your support <a name="support"></a>
+If you like this project rate a star⭐
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank microverse guiding me through this project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FAQ (optional) -->
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+- **Is this project work?**
+ - No, it is template project
+- **Is development finished?**
+ - Not yet.
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+## 📝 License <a name="license"></a>
+This project is [MIT](./MIT.md) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] You have the same issue in your content as in the menu, please 🙏🏽 add the spaces needed to look like the README template for professional documentation.

| Expected | Yours |
|---|---|
| ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/12da4761-2bde-4740-9895-3864ee3611ff) | ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/a2022c49-60e9-46c8-855c-5a838abcd8dc) |

Reply:
- [ ] Same here as pointed earlier :point_up: